src/components/Footer/styles.module.css

```diff
@@ -4,7 +4,7 @@
   padding-bottom: 5vh;
   height: 150%;
   text-align: center;
-  background-color: #f2f2f2;
+  background-color: rgb(251, 251, 251);
```

Comment:
In the spirit of "if the mountain won't come to Muhammad, then Muhammad must go to the mountain"... changing the background colour of the footer to match the background of the gif